config/share_notify.yml

```diff
@@ -0,0 +1,3 @@
+---
+host: <%= ENV.fetch('APPS_DHH_STAGING_SHARE_NOTIFY_HOST') {'https://staging.osf.io'}%>
```

Comment:
@VivianChu - ah, ok. so this is what's needed for the SHARE gem to work as-is? In that case, I'm 👍 for this now but noting that we should ideally refactor this, assuming @hweng can work with the other SHARE devs to make it possible. @hweng - should we setup a ticket in this Sprint for you to do that?